game/tests.py

```
@@ -2905,3 +2905,35 @@ def test_chess_puzzle_fen_validation(self):
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'coming_soon.html')
+        self.assertContains(response, "Leaderboard Coming Soon!")
+
+    def test_achievements_anonymous(self):
+        response = self.client.get(reverse('achievements'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'coming_soon.html')
+        self.assertContains(response, "Achievements Coming Soon!")
+
+    def test_leaderboard_authenticated(self):
+        User.objects.create_user(username='testuser', password='Password123!', email='testuser@example.com')
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)